src/vs/workbench/api/test/node/extHostSearch.test.ts

```diff
@@ -144,6 +145,17 @@ suite('ExtHostSearch', () => {
 					rpcProtocol,
 					new class extends mock<IExtHostInitDataService>() { override remote = { isRemote: false, authority: undefined, connectionData: null }; },
 					new URITransformerService(null),
+					new class extends mock<IExtHostConfiguration>() {
+						override async getConfigProvider() {
+							return {
+								getConfiguration() {
+									return {
+										get() { }
+									};
+								}
+							} as unknown as ExtHostConfigProvider;
```

Comment:
Is this cast necessary? Can we use `satisfies` here instead?

Reply:
Updated the mock config provider and removed the unknown cast. Thanks for spotting this!